src/ExpressionEvaluator/CSharp/Source/ExpressionCompiler/Symbols/DisplayClassVariable.cs

```diff
@@ -159,6 +159,8 @@ public override bool IsVolatile
                 get { return false; }
             }
 
+            internal override bool IsRequired => throw ExceptionUtilities.Unreachable;
```

Comment:
Could you please explain why this is the correct implementation here? I don't deeply understand the usage of this type but I'm concerned that when SymbolDisplay or some other API is updated to use 'required' that this code path might get hit.

Reply:
DisplayClass refers to lambda display classes, not SymbolDisplay. This should never be hit.